be/src/olap/rowset/segment_group.cpp

```diff
@@ -249,13 +249,13 @@ OLAPStatus SegmentGroup::add_zone_maps_for_linked_schema_change(
         return OLAP_SUCCESS;
     }
 
-    // 1. rollup tablet num_key_columns() will less than base tablet zone_map_fields.size().
+    // 1. rollup tablet get_num_zone_map_columns() will less than base tablet zone_map_fields.size().
     //    For LinkedSchemaChange, the rollup tablet keys order is the same as base tablet
-    // 2. adding column to existed table, num_key_columns() will larger than
+    // 2. adding column to existed table, get_num_zone_map_columns() will larger than
     //    zone_map_fields.size()
 
     int num_new_keys = 0;
-    for (size_t i = 0; i < _schema->num_key_columns(); ++i) {
+    for (size_t i = 0; i < get_num_zone_map_columns(); ++i) {
```

Comment:
All types of column has zone map now?

Reply:
float and double can generate zone map， bitmap cannot used in duplicate table, so it OK